packages/@ember/-internals/glimmer/tests/integration/components/target-action-test.js

```diff
@@ -1,12 +1,14 @@
+import { moduleFor, RenderingTestCase, ApplicationTestCase } from 'internal-test-helpers';
+
 import { assign } from '@ember/polyfills';
-import { moduleFor, RenderingTest, ApplicationTest } from '../../utils/test-case';
-import { strip } from '../../utils/abstract-test-case';
+import { strip } from 'internal-test-helpers';
```

Comment:
Shouldn't this get collapsed?

Reply:
yep, totally missed that. should be fixed now.